src/compiler/Stage/Emit/JavaScript.elm

```diff
@@ -20,6 +20,9 @@ emitExpr ( expr, _ ) =
         Literal (Int int) ->
             String.fromInt int
 
+        Literal (Float float) ->
+            String.fromFloat float
```

Comment:
Let's add some emit tests for this. We can try for a little bit more corner cases than with [Ints](https://github.com/elm-in-elm/compiler/blob/master/tests/EmitTest.elm#L42-L46) - eg. what does parsing `0.3` become? etc.

Then we'll be able to :heavy_check_mark: the emit tests in the README table.